containers/dotnet/definition-manifest.json

```diff
@@ -7,15 +7,12 @@
 		"tags": [
 			"dotnet:${VERSION}-${VARIANT}"
 		],
-		"architectures": {
-			"6.0-focal": ["linux/amd64", "linux/arm64"],
-			"6.0-bullseye-slim": ["linux/amd64", "linux/arm64"],
-			"5.0-focal": ["linux/amd64", "linux/arm64"],
-			"5.0-bullseye-slim": ["linux/amd64", "linux/arm64"],
-			"3.1-focal": ["linux/amd64", "linux/arm64"],
-			"3.1-bullseye": ["linux/amd64", "linux/arm64"]
-		},
+		"architectures": ["linux/amd64", "linux/arm64"],
 		"variantTags": {
+			"6.0-focal": [
+				"dotnet:${VERSION}-6.0",
```

Comment:
my guess is only one of these lines is needed, since focal doesn't appear to have dotnetcore variant ?